examples/__testfixtures__/styled-components-to-ucl/interpolated.output.ts

```diff
@@ -1,14 +1,18 @@
 import { Box } from '@rbilabs/universal-components';
 
-const MAX_CONTENT_WIDTH_PX_DESKTOP = 12;
+const WIDTH_PX = 12;
+const HEIGHT_PX = 12;
 
 export const InterpolatedVariables = Box.withConfig({
-  width: `${MAX_CONTENT_WIDTH_PX_DESKTOP}px`,
+  width: `${WIDTH_PX}px`,
+  height: `pre${HEIGHT_PX}rem`,
```

Comment:
Should we be concerned about this?

Why we should we allow the ‘pre’ in there? And then should we comment it out if it includes rem and require people to manually convert?

Reply:
`pre` is probably a bad example. I think we are doing thing like `-${HEIGHT_PX}px`, though

For `rem` yes, maybe we should just error on any `post` that isn't `px`?

Reply:
👍

Reply:
I'm going to merge and we can iterate